app/main.py

```diff
@@ -1 +1,24 @@
-# write your code here
+import os
+
+
+def move_file(command: str) -> None:
+    command_name, file_to_move, new_file = command.split()
```

Comment:
first you have to check if you can do the baking, i.e. if the length of `command.split()` is equal to three